test/mocks/MockAVS.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: BSD 3-Clause License
+pragma solidity ^0.8.24;
+
+contract MockAVS {
```

Comment:
You can also use lib/eigenlayer-contracts/src/test/mocks/MockAVSRegistrar.sol which is the one used by eigen to mock an avs